test/kit_pid_helpers.cpp

```diff
@@ -8,40 +8,24 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
-#include "kit_pid_helpers.hpp"
-
-#include <wsd/COOLWSD.hpp>
+#include <kit_pid_helpers.hpp>
 
 #include <set>
 #include <chrono>
 #include <iostream>
 #include <thread>
 #include <string>
 
-#include <Poco/DirectoryIterator.h>
-#include <Poco/FileStream.h>
-#include <Poco/StreamCopier.h>
+#include <wsd/COOLWSD.hpp>
 
-#include <Common.hpp>
-#include "Util.hpp"
-#include "lokassert.hpp"
-#include "test.hpp"
-#include "testlog.hpp"
+#include <lokassert.hpp>
+#include <testlog.hpp>
 
-std::set<pid_t> kit_pid_helpers::getKitPids()
```

Comment:
If we needed a namespace, and we don't here. It would be cleaner to define the namespace foo { and define functions inside it I think }

Reply:
The other functions were in the `helpers` namespace, but have `using namespace helpers;` in the test files. I was trying to avoid putting functions in the helpers namespace that aren't defined in helpers.hpp, but I'm not sure if that's actually an issue.

So should I:
- Move these to the helpers namespace
- Keep them in their own (probably renamed to PidHelpers)
- No namespace at all (I don't like this, given there are other functions with the same name in different namespaces, like COOLWSD::getKitPids)

For the implementation in the cpp file, I suppose you could implement them inside brackets as `namespace foo { void bar(){...} }`, but I have never seen it done that way, I have always seen it as `void foo::bar(){...}`. Please confirm this is what you want, I would do it but it feels weird. Maybe I'm confused and it is only necessary for methods?

And then in tests, if using a new namespace, should I:
- `using namespace PidHelpers;`, and call getKitPids()
- or call PidHelpers::getKitPids()

These are style questions, so your call.

Reply:
on namespaces: cf. $ git grep 'namespace' | grep cpp # :-)
No problem to keep helpers namespace for now, or use PidHelpers if it makes you happy.
Re: invocation - we like the code to be short & readable, without overly-cutsey naming as a general rule.